VMM/Templates/NC/NCSetup.cr/PrepareNodeForNetworkController.ps1

```diff
@@ -25,25 +25,6 @@ try
     #------------------------------------------    
     Log "Installing NetworkController Role.."
     Add-WindowsFeature -Name NetworkController -IncludeManagementTools
-
-
-
-
-    #------------------------------------------
-    # Disable Restart Manager in NC for Windows Server RTM
-    #------------------------------------------    
-    $manifestPath = ([System.IO.Directory]::GetParent([System.Environment]::SystemDirectory)).ToString()
-    $manifestPath += "\NetworkController\TemplateClusterManifest.xml"
-
-    [xml] $ncClusterManifest = Get-Content $manifestPath
-    $ncClusterManifestVersion = $ncClusterManifest.ClusterManifest.Attributes["Version"].Value
-
-    if($ncClusterManifestVersion -eq "10.1.0.0")

```

Comment:
Are you sure we should remove this check? No one should be on pre-9D patch but if they are, this check helps catch it.